cloud-etl/docs/index.rst

```diff
@@ -114,12 +105,31 @@ Local Tools
 -  ``python``
 -  Download `Confluent Platform <https://www.confluent.io/download/>`__ |release|: for more advanced Confluent CLI functionality (optional)
 
+====================
+Cost to Run Tutorial
+====================
+
+Caution
+~~~~~~~
+
+.. include:: ../../ccloud/docs/includes/ccloud-examples-caution.rst
+
+This example also uses real cloud resources, including:
+
+* AWS Kinesis or RDS PostgreSQL
+* One of the cloud storage providers, depending on your configuration
```

Comment:
The latter point seems vague given that the first point is specific. Are you suggesting Confluent Cloud in GCP/Azure/AWS or is this referring to general costs of GCP/Azure/AWS like data transfer costs, vpc costs, etc in those cloud providers?

Reply:
Updated in latest commit, WDYT?

Reply:
That seems a lot more clear imo :) thank you!